large_page-c/lp_preload.c

```diff
@@ -37,6 +39,7 @@ static int tryMapAllDSOs(struct dl_phdr_info* hdr, size_t size, void* data) {
 
 void __attribute__((constructor)) map_to_large_pages() {
   bool is_enabled = true;
+  printf("TID: %d\n:", gettid());
```

Comment:
Please use fprintf() instead of printf() for consistency.

Reply:
@uttampawar , Sorry for the late update as I did not get the email notification of your comment. I just updated the PR to address your comment.